src/app.ts

```diff
@@ -670,14 +684,15 @@ const awsx = (): void => {
     })
     .command({
       command:
-        'add-profile [profile] [access-key] [secret-key] [default-region] [output-format] [mfa-arn] [mfa-expiry]',
+        'add-profile [profile] [access-key] [secret-key] [secret-key-expiry] [default-region] [output-format] [mfa-arn] [mfa-expiry]',
```

Comment:
For existing `profiles` added command to update profile with [secret-key-expiry] field